### DISCUSSION TITLE
Twinny Not Recognizing the Context

### DISCUSSION THREAD

**Discussion by lochana** [rjmacarthy/twinny#409] [Q&A]
**I have set llama3.1 for chat and qwen-coder2.5 for FIM.**

**With the following custom template:**

Correct the grammar and style mistakes in the following text. Do not change the meaning:
{{{text}}}
Output should only include revised text in paragraph form. Do not explain the changes made in your response. 

 **I get**

Please go ahead and provide the text that needs correction. I'll make sure to revise it without altering its meaning, while keeping it concise and clear.

**I can clearly see the context selected, but it does not recognize it. However, it does work once I click the regenerate button.**

**Comment by rjmacarthy** [comment]
Hello try {{{code}}} instead of text.
